server/src/routes/v1/cards.js

```diff
@@ -0,0 +1,5 @@
+const cardsController = require('../../controllers/v1').cards
+
+module.exports = (api) => {
+  api.post('/cards', cardsController.create)
```

Comment:
@nikitasakov can't we use `api.post('/', cardsController.create)` instead?

As I understand that's the advantage of express routing

Reply:
@AleksSenkou Nope, he does not see this url 😔. 
<img width="763" alt="2018-12-03 16 04 52" src="https://user-images.githubusercontent.com/32716983/49375536-a508ee80-f715-11e8-8a33-110a0a362d46.png">

Reply:
@nikitasakov ok, let's go with what we have for now